Packages/MovieDBData/Sources/MovieDBData/Utils/Extentions/Sequence+Extention.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Sequence+Extention.swift
+//  MovieDB
+//
+//  Created by Mouad Bj on 16/3/2023.
+//
+
+import Foundation
+
+extension Sequence where Element: Hashable {
```

Comment:
Yes 👍🏽, I dragged it to Core package while refactoring, but forgot to delete it from the Data module. in an actual project, we must have something like sonar to avoid these mistakes.